packages/messaging-interop-types/package.json

```
@@ -0,0 +1,25 @@
+{
+    "name": "@firebase/messaging-interop-types",
+    "version": "0.1.0",
```

```suggestion
    "version": "0.0.1",
```

we will have a changeset to bump minor, so it will be released as `0.1.0`